src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(value) {
```

Comment:
The adder function only takes a single argument. It needs to be modified to accept multiple arguments.